content/intro-to-storybook/react/en/simple-component.md

```diff
@@ -93,7 +93,7 @@ To tell Storybook about the component we are documenting, we create a `default`
 - `excludeStories` -- exports in the story file that should not be rendered as stories by Storybook.
 - `argTypes` -- specify the [args](https://storybook.js.org/docs/react/api/argtypes) behavior in each story.
 
-To define our stories, we export a function for each of our test states to generate a story. The story is a function that returns a rendered element (i.e. a component with a set of props) in a given state---exactly like a [Stateless Functional Component](https://reactjs.org/docs/components-and-props.html).
+To define our stories, we export a function for each of our test states to generate a story. The story is a function that returns a rendered element (i.e. a component with a set of props) in a given state.
```

Comment:
@tai2 we have a bit of room for improvement here. Instead of dropping the sentence altogether, we only need to remove stateless and leave the rest as is.

The reasoning behind this is that we technically are implementing functional components with our stories and for someone that might be starting with React and Storybook and doesn't know what are functional components we offer the link so that they can follow up and read more about it.

Let me know what do you think?

Reply:
@jonathanherdt 

Make sense. We need to explain what is the function components because Storybook's argument mechanism relies on the JS function. 

https://reactjs.org/docs/components-and-props.html#function-and-class-components
Do you think the above link is enough for it? They're referring to "function components" a little.

Reply:
@tai2, I think that you pinged the wrong person. It's jonniebigodes not jonathanherdt 😛! But no worries. Regarding the link, we could use it. It gives an overview of what a component is. Functional or Class based one. It's perfect for people just starting with React or React+ Storybook. For more knowledgeable readers that are more familiar with React and Functional components they understand the context and move on.

Let me know and we'll go from there

Reply:
@jonniebigodes 

Sorry for mistaking your name 😅 

OK, I just changed from "Stateless Functional Component" to "function component" then.

https://github.com/chromaui/learnstorybook.com/pull/470/commits/b31e1502aef7f7c2a73aa5faf7259490a2a09b0a